_includes/store/helsinkisystems-lexoffice.md

```diff
@@ -0,0 +1,18 @@
+## Description
+
+Lexoffice plugin for Kimai is a plugin that allows you to automatically create invocies in Lexoffice, whenver an invoice in Kimai is created. By using the Lexoffice API (API key is needed) this plugin can create invoice drafts and finalized invoices in Lexoffice, based on your settings. Furthermore, this plugin allows you to customize the title and description of the items in the Lexoffice-invoice. The plugin therefore offers a few replacers you can choose from and that are replaced dynamically when creating the invoice for Lexoffice.
+
+## Installation
+
+This plugin does not need a database table, so there is no installation of the plugin necessary.
+To communicate with Lexoffice however, you need to install the Lexoffice-php-sdk composer package:
+``` composer require helsinki-systems/lexoffice-php-sdk:dev-master ```
+
+UPDATE WARNING: As this plugin uses composer it breaks on Kimai Updates.
+You will have to rerun the composer installation step after each update. 
```

Comment:
I think this is still not clear / I doubt that regular users will understand that. 
Maybe you can create `include/composer-update-warning.md` and add something like:

UPDATE WARNING: The next time you update Kimai you will see something like ... (show the merge message that will likely appear) what happens and you have to revert your changes first, then update Kimai and afterwards reinstall the plugin. Please read https://www.kimai.org/documentation/updates.html#changed-files

Then I can re-use this include on the LDAP docu page as well and we can update the shown warning on all related pages.